### ISSUE TITLE
JOHNNIAC Simulator on Bitsavers

### ISSUE THREAD

**Issue by PashPaw** [sebras/JOSS-I#17]
It looks like Al Kossow uploaded a JOHANNIAC simulator on Bitsavers today.  It looks like the binaries are for DOS and has something about JACASS in it and with some source code.

Todo:  Try it on DOSBox.  There's no real documentation here except for JACASS in an obscure format.

**Comment by larsbrinkhoff** [comment]
I think it was written by Mort Bernstein.  I checked, but I haven't found any contact info.

**Comment by larsbrinkhoff** [comment]
Oh, and a link to the simulator: http://bitsavers.org/bits/Rand/JOHNNIAC/

**Comment by larsbrinkhoff** [comment]
*JOHNNIAC simulator and Computer History Museum lecture*

https://www.computerhistory.org/collections/catalog/102734545

**Comment by PashPaw** [comment]
Thanks!  It does seem like a rudimentary simulator but it’s likely useful for the documentation for JACASS.  

It likely isn’t wholly usable for our purposes.  It does run in DOSBox though.  I just haven’t figured it out beyond starting it in DOSBox.

**Comment by larsbrinkhoff** [comment]
But it doesn't come with much useful JOHNNIAC software, does it?

I did see some kind of bootstrapping punched card, so that's nice.

**Comment by sebras** [comment]
Yes, Al Kossow made it available because I asked about JOHNNIAC documentatio. I also saw the J267 loader which has previously been missing. :) The author of the simulator is not the one @larsbrinkhoff mentioned. Many of the other pieces of DOS software are just to get aquainted with the environment it seems.

**Comment by larsbrinkhoff** [comment]
Here is newer emulator which has made good progress:  
https://github.com/Epigenetic/johnniac_emulation  
https://epigeneticslab.net/
